libraries/AP_RangeFinder/AP_RangeFinder_LeddarOne.cpp

```diff
@@ -130,33 +161,35 @@ bool AP_RangeFinder_LeddarOne::CRC16(uint8_t *aBuffer, uint8_t aLength, bool aCh
  */
 LeddarOne_Status AP_RangeFinder_LeddarOne::send_request(void)
 {
-    uint8_t data_buffer[10] = {0};
-    uint8_t i = 0;
+    uint8_t send_buffer[10] = {0};
```

Comment:
the for loop goes 8 times but the index is size 10? There is 6 for data and 2 for CRC. Shouldn't this be [8]?